src/services/services.ts

```diff
@@ -1958,6 +1958,12 @@ namespace ts {
                     (char >= CharacterCodes.A && char <= CharacterCodes.Z) ||
                     (char >= CharacterCodes._0 && char <= CharacterCodes._9);
             }
+
+            function isNodeModulesFile(path: string): boolean {
+                const node_modulesFolderName = "/node_modules/";
+
+                return path.indexOf(node_modulesFolderName) !== -1;

```

Comment:
From the point of readability `path.indexOf(node_modulesFolderName) !== -1` sucks.

As with any double-negative, you're forced to pause and do some mental calculations: *"ah, it's checking for position, then asserts that position is not good; no wait it asserts that it's not wrong, then proceeds — therefore this statement checks that the substring exists in the path string! Phew...."*

Rules are academic. In the end what counts is cost of maintenance. Replacing `!==-1` with `>=0` improves readability and reduces possibility of bugs.